core/trino-main/src/main/java/io/trino/sql/analyzer/QueryExplainer.java

```diff
@@ -29,6 +29,7 @@
 import io.trino.sql.planner.TypeAnalyzer;
```

Comment:
Update commit message to
```
Add ability to anonymise output of PlanPrinter

This can be used to collect anonymised plans through
query event listeners and to print anonymized plans
from EXPLAIN.
```